client/src/pages/Resource/index.jsx

```diff
@@ -0,0 +1,78 @@
+import { useEffect, useState } from "react";
+import { useParams } from "react-router-dom";
+
+import { ResourceService, useService } from "../../services";
+
+import "./Resource.scss";
+
+export default function Resource() {
+	const [resource, setResource] = useState(undefined);
+	const resourceService = useService(ResourceService);
+	const { id } = useParams();
+
+	useEffect(() => {
+		resourceService.getOne(id).then(setResource);
+	}, [id, resourceService]);
+
+	if (!resource) {
+		return null;
```

Comment:
If the component returns `null` if the resource is not loaded, it might lead to a flash of no content. 

How about implementing a loading state to show a spinner or placeholder while the resource is being fetched?